packages/pymedphys_logfiles/src/pymedphys_logfiles/elekta/bygantry.py

```diff
@@ -148,13 +141,17 @@ def calc_logfile_mu_density_bygantry(index, config, logfile_group,
 
 
 def compare_logfile_group_bygantry(index, config, cursor, logfile_group,
```

Comment:
```suggestion
def compare_logfile_group_by_gantry_angle(index, config, cursor, logfile_group,
```

Reply:
This change would need to be propagated... so can't just accept this here